### DISCUSSION TITLE
Execute steps on workflow rejection

### DISCUSSION THREAD

**Discussion by jjacob27** [community/community#4546] [Actions]
Hi,

Also posted here: https://github.community/t/running-steps-on-workflow-reject/188603

**My request:** 
I have a workflow that requires manual approval. I modeled this using environments and environment protection rules as detailed in the documentation at https://docs.github.com/en/actions/managing-workflow-runs/reviewing-deployments

Now when a deployment is rejected by the approver, I need to run a couple of steps such as “send an email” or do some cleanup. 

Thanks

**Comment by ethomson** [comment]
Thanks for the feedback - I can imagine that we can provide you more information about the `workflow_run`'s `conclusion` in this case.  I'll make sure that we add this to the backlog.

**Comment by jaroslaw-rola** [comment]
Hello, 
I have a question - was this feature requested by @jjacob27 added to Actions?
I mean if there is a possibility to trigger workflow on rejected deployment?

**Comment by luisiturrios1** [comment]
Hello. 
I am making a GitHub workflow for canary deployment but I can´t find any way to execute steps when deployment is rejected

**Comment by david-dest01** [comment]
any updates here?

**Reply by dibyadhar** [reply]
Has anyone found any alternative ?

**Reply by aadi1011** [reply]
I do not think there is any alternative within GitHub for this unfortunately. I would like to have a solution for the same.
